adapters/repos/db/inverted/prop_length_tracker.go

```diff
@@ -61,6 +61,7 @@ func NewPropertyLengthTracker(path string) (*PropertyLengthTracker, error) {
 
 	stat, err := f.Stat()
 	if err != nil {
+		f.Close()
```

Comment:
**suggestion:** if there is an error the file shall not be opened to be closed 🤔  

we can defer `f.Close()` in L61 so it would be closed once this func returns